Workflow/Sources/WorkflowHost.swift

```diff
@@ -36,7 +36,7 @@ public final class WorkflowHost<WorkflowType: Workflow> {
 
     private let (outputEvent, outputEventObserver) = Signal<WorkflowType.Output, Never>.pipe()
 
-    private let rootNode: WorkflowNode<WorkflowType>
+    let rootNode: WorkflowNode<WorkflowType>
```

Comment:
You mentioned concerns about potentially leaking some Workflow internals that could lead to Workflows hanging around longer than expected. Is this one of those places?

Reply:
Sometimes I use this little trick to make it clearer why something that one may think should be private isn't, and also to emphasize that while it isn't private, it's also no public:

```swift
// @testable
internal let rootNode: WorkflowNode<WorkflowType>
```

Reply:
You could also leverage `@_spi` here to force someone to opt into the SPI interface to access this.